.travis.yml

```diff
@@ -1,9 +1,6 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
-  - "4"
-  - "5"
   - "6"
-  - "7"
+  - "8"
+  - "10"
```

Comment:
Based on the [official Node release schedule](https://github.com/nodejs/Release/blob/643af71ee8365efbdccaa134690425ea05931279/README.md#release-schedule), Node 4 reached EOL in April 2018, and currently the odd-numbered "development" versions of Node are also EOL, hence 6, 8, and 10 are the currently supported LTS versions.